cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -21,8 +22,8 @@
     {%- endfor %}
 
     {% for block in page.content -%}
-        {% if block.block_type == 'feedback' %}
-            {% include 'setup-feedback-and-render.html' %}
+        {% if block.block_type in ['feedback', 'conference_registration_form'] %}
```

Comment:
If there's such a thing as `getattr` in jinja2 then maybe. This will eventually be refactored.